docs/en/stack/ml/nlp/ml-nlp-e5.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[ml-nlp-e5]]
+= E5 – EmbEddings from bidirEctional Encoder rEpresentations
+++++
+<titleabbrev>E5</titleabbrev>
+++++
+
+:frontmatter-description: E5 is a multilingual dense vector model.
+:frontmatter-tags-products: [ml] 
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [analyze]
+
+experimental[]
+
+EmbEddings from bidirEctional Encoder rEpresentations - or E5 -  is a {nlp} 
+model that enables you to perform multi-lingual semantic search by using dense 
+vector representations. This model performs best for non-English language 
+documents and queries. If you want to perform semantic search on English 
```

Comment:
> This model performs best for non-English language documents and queries

I can't say if it performs _best_ or not, can you soften this to say this is the recommended model for non-English